ngx-http-concat.php

```diff
@@ -114,8 +114,19 @@ function concat_get_path( $uri ) {
 	concat_http_status_exit( 400 );
 
 // array( '/foo/bar.css', '/foo1/bar/baz.css' )
-if ( 0 == count( $args ) || count( $args ) > $concat_max_files )
+if ( 0 == count( $args ) || count( $args ) > $concat_max_files ) {
+	if ( count( $args ) > $concat_max_files ) {
+		trigger_error(
+			sprintf(
+				'Cannot concatenate over %d files in %s',
+				$concat_max_files,
+				$_SERVER['REQUEST_URI']
```

Comment:
Would this actually be ngx-http-concat.php's URI?

Reply:
Error I captured in a sandbox environment (truncated middle for readability) :

```
PHP message: PHP Warning: Cannot concatenate over 150 files in /_static/??-eJy...Nv6Bufg== in /var/www/wp-content/mu-plugins/http-concat/ngx-http-concat.php on line 120
```

Reply:
@trepmal in that case is the url helpful in any meaningful way in terms of providing information for debug?

Reply:
there are several cases which could result in a 404. I think it's helpful to have the info just in case there are multiple issues.

#78 could make this moot though

Reply:
Yes, I think with #78 merged this may be closed